user_tools/src/spark_rapids_pytools/resources/qualification-conf.yaml

```diff
@@ -233,11 +235,14 @@ local:
       resultColumnName: 'Unsupported Operators Stage Duration'
       percentResultColumnName: 'Unsupported Operators Stage Duration Percent'
     topCandidates:
-      categoryColumnName: 'Estimated GPU Speedup Category'
+      categoryColumnName: 'Estimated GPU Speedup Category**'
       outputColumns:
         - 'App ID'
         - 'App Name'
-        - 'Estimated GPU Speedup Category'
+        - 'Estimated GPU Speedup Category**'
```

Comment:
We might have to cleanup headers as `**` would show up in the summary CSV file as well.

Reply:
good point I'll investigate, maybe just try to add in code

Reply:
change to just add to the stdout in the code and the yaml stays without the **